### PR TITLE
{vis}[foss/2020b] Nilearn v0.7.1 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
@boegelbot Please test @ generoso

File: easybuild/easyconfigs/n/Nilearn/Nilearn-0.7.1-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+name = 'Nilearn'
+version = '0.7.1'
+
+homepage = 'https://nilearn.github.io/'
+description = """Nilearn is a Python module for fast and easy statistical learning on NeuroImaging data."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+sources = [SOURCELOWER_TAR_GZ]
```
Comment (branfosj) [inline easybuild/easyconfigs/n/Nilearn/Nilearn-0.7.1-foss-2020b.eb:11]: please add the checksum

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/n/Nilearn/Nilearn-0.7.1-foss-2020b.eb (hunk 1 above)
Comment (akesandgren) [reply easybuild/easyconfigs/n/Nilearn/Nilearn-0.7.1-foss-2020b.eb:11]: Done

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.4
See https://gist.github.com/e87095672e3501d128740ca3c59034d3 for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
@branfosj could you confirm you're happy with the changes? I don't like to dismiss someone else's review...

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
> @branfosj could you confirm you're happy with the changes? I don't like to dismiss someone else's review...

Yes, I am happy with the changes.

**Comment by verdurin** [comment]
Going in, thanks @akesandgren!

**Pull request merged**